lib/versions.js

```diff
@@ -43,7 +43,7 @@ function normalizeVersion(release) {
         .value();
 
     return {
-        tag: normalizeTag(release.tag_name),
+        tag: normalizeTag(release.tag_name).split('-')[0],
```

Comment:
This had to do with an issue I was running into with the new `update` endpoints if `tag` had a channel suffix, like `v0.0.1-alpha`. This was awhile back and I don't exactly remember so I'll investigate it ASAP and try to see why I needed this ... if only I had wrote tests @joeferraro :cry: